### ISSUE TITLE
Avoid unnecessary refresh

### ISSUE THREAD

**Issue by antonellopasella** [binaryage/xrefresh#9]
Great plugin!

Is there a way to avoid refresh when I save a file not connected with the current page?
If I edit a js not included in the current page, why reload the page?
If I edit a php that respond to an ajax call, why reload the whole page?

I know that there are problems with mod_rewrite structure and so on, but would be great for some environments:D

Tkz a lot!

**Comment by darwin** [comment]
Yes I see, would be nice to add this "strict mode". I can image checker under XRefresh disclosure arrow in firebug menubar (under Soft CSS Refresh and Soft JS Refresh checkers)

**Comment by NV** [comment]
I've no idea how to implement this. But I'd like to!

It's easy to achieve for `file://localhost`, but what's about `http://example.local/`? What's about a dynamic content?
